accounts-db/src/accounts_db.rs

```diff
@@ -10593,8 +10594,7 @@ pub mod tests {
         }
     }
 
-    #[test]
-    fn test_accountsdb_scan_account_storage_no_bank() {
+    fn accountsdb_scan_account_storage_no_bank_test_case<T: WritableAccountsFileProvider>() {
```

Comment:
@brooksprumo: And, the original test function will now take a generic to allow it to run with different AccountsFile formats.